Dockerfile

```diff
@@ -1,3 +1,3 @@
 FROM nginx:alpine
-COPY ./dist/compressed /usr/share/nginx/html/beta
```

Comment:
this needs to dynamic based on type of release

Reply:
Dockerfile changed to dynamically load nginx location.